c2corg_ui/static/partials/feed.html

```diff
@@ -28,9 +28,9 @@ <h4 class="text-success text-center" translate ng-show="feedCtrl.noFeed">This us
       </a>
     </div>
   </article>
-
-  <article class="feed-list-item" ng-repeat="doc in feedCtrl.documents track by doc.id"
-           ng-init="type = feedCtrl.getDocumentType(doc.document.type);">
+  
+  <article class="feed-list-item flex" ng-repeat="doc in feedCtrl.documents track by doc.id"
+           ng-init="type = feedCtrl.documentType(doc.document.type);">
```

Comment:
This function is actually named ``getDocumentType``. Is your branch up-to-date?
https://github.com/c2corg/v6_ui/blob/master/c2corg_ui/static/js/feed.js#L220